examples/basic-usage/server.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+const passport = require('passport');
+const { Strategy } = require('@superfaceai/passport-twitter-oauth2');
+const session = require('express-session');
+require('dotenv').config();
+
+passport.serializeUser(function (user, done) {
+  done(null, user);
+});
+passport.deserializeUser(function (obj, done) {
+  done(null, obj);
+});
+
+// Use the Twitter OAuth2 strategy within Passport
+passport.use(
+  new Strategy(
+    {
+      clientID: process.env.TWITTER_CLIENT_ID,
+      clientSecret: process.env.TWITTER_CLIENT_SECRET,
+      clientType: 'confidential',
+      callbackURL: `${process.env.BASE_URL}/auth/twitter/callback`,
+    },
+    // <3> Verify callback
```

Comment:
question: I did not find 1, 2, 6 steps. Am I missing something? Also this is not consistent with the second example where you do not have numbered steps.

Reply:
Sorry, these were leftovers from my tutorial, I will remove the numbers everywhere.